src/benchmark.cc

```diff
@@ -115,7 +115,13 @@ std::string* GetReportLabel() {
 
 // Global variable so that a benchmark can report an error as a human readable
 // string. If error_message is null no error occurred.
-static std::atomic<const char*> error_message = ATOMIC_VAR_INIT(nullptr);
+#if defined(BENCHMARK_WORKAROUND_VS_2013)
```

Comment:
I was concerned about exposing other compilers to the _MSC_VER check; will
```
#if defined(_MSC_VER) && _MSC_VER <= 1800
```
work?